### ISSUE TITLE
Translation in French

### ISSUE THREAD

**Issue by m-hefti** [merlos/iOS-Open-GPX-Tracker#175]
I've translated the files. As some strings might be longer in French than the English version, I don't know how the UI will handle it. Let me know if some shortage is needed. I also might maintain the translations.
Thanks for the app!
[OpenGpxTracker.zip](https://github.com/merlos/iOS-Open-GPX-Tracker/files/4894052/OpenGpxTracker.zip)

**Comment by vincentneo** [comment]
Hi @m-hefti 
Apologies for not noticing this issue. We should get this going, real soon.

**Comment by vincentneo** [comment]
Hi @m-hefti, 

Most length seems fine, with the exception of some like:

The Force English language date format string
<img width="202" alt="Screenshot 2020-07-19 at 6 17 23 PM" src="https://user-images.githubusercontent.com/23420208/87872660-3ad9b600-c9ed-11ea-90e3-e286ea3d00d5.png">
But in actuality, it seems like even the English version is not coping well with that, so maybe it would just need a font size reduction.

Another, is the save and reset button.
<img width="187" alt="Screenshot 2020-07-19 at 6 26 04 PM" src="https://user-images.githubusercontent.com/23420208/87872702-986e0280-c9ed-11ea-9792-ed098b69ab23.png">
While it is readable, it seems to be too small, quite hard to read easily, as it has to accommodate the length of the text, and is offset a little down too.
It would be great if this can be shorten. 

Thanks for the translation!

**Comment by m-hefti** [comment]
Thanks for the screenshots. I've updated the translations regarding your comments.
To solve the length of the first string, I changed the wording to "Utiliser le format anglais?" ("Use English format?")
I've updated the "Save" and "Reset" translations both in:
- /OpenGpxTracker/fr.lproj/Localizable.strings
- /OpenGpxTracker-Watch/fr.lproj/Interface.strings
as I don't understand which one is actually used in the UI.
[OpenGpxTracker.zip](https://github.com/merlos/iOS-Open-GPX-Tracker/files/4944042/OpenGpxTracker.zip)
